src/plugins/plugins.h

```diff
@@ -206,4 +208,18 @@ class drakvuf_plugins
 
 /***************************************************************************/
 
+// Retrieves system time in seconds and microseconds.
+inline timeval get_time()
```

Comment:
`static` perhaps?

Reply:
Marking global function `static` makes it inaccessible to other translation units. This makes sense in *.cpp files, but this one is in *.h, i.e., supposed to be included into different translation units. As far as I understand marking it `static` wouldn't have any practical effect.